src/content/docs/nrql/nrql-tutorials/nrql-tutorial-advanced-functions.mdx

```diff
@@ -101,7 +101,7 @@ With this, you've now learned all aggregation functions that curently exist in N
 
 ### Use advanced math functions
 
-NRQL also supports some nifty, more advanced mathetmatical functions for those who need to get really deep into their data. These functions can extrapolate values to power of X, find square roots, apply clamping to impose upper and lower bounds, or even just keep values postive on the Y axis.
+NRQL also supports more advanced mathetmatical functions for those who need to get really deep into their data. These functions can extrapolate values to power of X, find square roots, apply clamping to impose upper and lower bounds, or even just keep values postive on the Y axis.
```

Comment:
```suggestion
NRQL also supports more advanced mathematical functions for those who need to get really deep into their data. These functions can extrapolate values to power of X, find square roots, apply clamping to impose upper and lower bounds, or even just keep values postive on the Y axis.
```

Reply:
I wish it was spelled with the extra t, but I have fixed it!